src/valkey-benchmark.c

```diff
@@ -1870,6 +1904,9 @@ int parseOptions(int argc, char **argv) {
         "                    Note: If --cluster is used then number of clients has to be\n"
         "                    the same or higher than the number of nodes.\n"
         " -n <requests>      Total number of requests (default 100000)\n"
+        " --duration <seconds>\n"
+        "                    Run benchmark for specified number of seconds (mutually exclusive with -n)\n"
+        " --warmup <seconds> Run benchmark for specified warmup period before recording data\n"
```

Comment:
We can keep the output within 80 columns (the default terminal size).

Let's use a period at least when it's a full sentence.

```suggestion
        " --duration <seconds>\n"
        "                    Run benchmark for the specified number of seconds. Mutually\n"
        "                    exclusive with -n.\n"
        " --warmup <seconds> Run benchmark for the specified warmup period before\n"
        "                    recording data.\n"
```